src/server/ApiManager.js

```diff
@@ -0,0 +1,96 @@
+        this.apiUrlMap.set('low-bikes', 'low-high-bikes-products');
+        this.apiUrlMap.set('high-components', 'high-low-components-products');
+        this.apiUrlMap.set('low-components', 'low-high-components-products');
+    }
+    
+    getCategoryUrl(categoryId, sort) {
+
+        var apiUrlKey = sort + '-' + categoryId;
+        return this.apiUrlEndpoint + this.apiUrlMap.get(apiUrlKey) + '.json';
+
+    }
+    */
+
+    /* Campmor API */
+    constructor() {
+
+        this.apiUrlEndpoint = 'https://campmor.ampify.wompmobile.com/campmor';
+        this.apiCategoriesEndpoint = this.apiUrlEndpoint + '/fetchCategories';
+        this.apiProductEndpoint = this.apiUrlEndpoint + '/fetchProduct';
+
+        this.apiUrlMap = new Map();
+
+        //Men's Shirts: https://campmor.ampify.wompmobile.com/campmor/fetchCategories?categoryId=200368502&sortBy=priceLtoH
+        this.apiUrlMap.set('men-shirts-high-low', 'categoryId=200368502&sortBy=priceHtoL');
+        this.apiUrlMap.set('men-shirts-low-high', 'categoryId=200368502&sortBy=priceLtoH');
+
+        //Men's Shorts: https://campmor.ampify.wompmobile.com/campmor/fetchCategories?categoryId=200368503&sortBy=priceLtoH
+        this.apiUrlMap.set('men-shorts-high-low', 'categoryId=200368503&sortBy=priceHtoL');
+        this.apiUrlMap.set('men-shorts-low-high', 'categoryId=200368503&sortBy=priceLtoH');
+
+
+        //Men's Shorts: https://campmor.ampify.wompmobile.com/campmor/fetchCategories?categoryId=200368509&sortBy=priceLtoH
+        this.apiUrlMap.set('women-shirts-high-low', 'categoryId=200368507&sortBy=priceHtoL');
+        this.apiUrlMap.set('women-shirts-low-high', 'categoryId=200368507&sortBy=priceLtoH');
+
+        //Men's Shorts: https://campmor.ampify.wompmobile.com/campmor/fetchCategories?categoryId=200368507&sortBy=priceLtoH
+        this.apiUrlMap.set('women-shorts-high-low', 'categoryId=200368509&sortBy=priceHtoL');
+        this.apiUrlMap.set('women-shorts-low-high', 'categoryId=200368509&sortBy=priceLtoH');
+
+    }
+
+    getCategoryUrl(categoryId, sort) {
+
+        var apiUrlParams = categoryId + '-' + sort;
+        return this.apiCategoriesEndpoint + '?' + this.apiUrlMap.get(apiUrlParams);
+
+    }
+
+    parseCategory(apiCategoryResponse) {
+
+        let prodCategory = JSON.parse(apiCategoryResponse);
+        let prodListing = prodCategory.matchingProducts;
+
+        var parsedCategory = {
+            items: []
+        };
+
+        for (var prod of prodListing) {
+            let originalProd = prod.Value;
+            let parsedProd = new Object();
+            parsedProd.productId = originalProd.Main_Id;
+            parsedProd.name = originalProd.Product_Title;
+            parsedProd.description = originalProd.Product_Title; /* Missing field on Campmor API */
+            parsedProd.price = originalProd.Price;
+            parsedProd.image = originalProd.Photo;
+            parsedProd.category = originalProd.Main_Id; /* Missing field on Campmor API */
+
```

Comment:
It's nice to restore those missing fields if we think we'll need them!  Which I don't know :)